src/flowfield.py

```diff
@@ -115,7 +118,7 @@ def draw(self,
                                                '<br>y = %{y:.4f}'+
                                                '<br>u = %{z:.4e}'+
                                                '<extra></extra>', ## '<extra></extra>' removes the trace name from hover text
-                                 colorbar=dict(title_text= LONG_NAME_DICT["xvel"] + '   [m s^-1]',
+                                 colorbar=dict(title_text= LONG_NAME_DICT["xvel"] + '   [m s<sup>-1</sup>]',
```

Comment:
This still seems like a suboptimal solution, but I was also not able to find a solution.

Reply:
Yea... seems to be a Plotly-Streamlit issue atm